content/en/blog/2024/2024-03-29-first-vote-results.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "First vote results"
+linkTitle: First vote results
+date: 2024-03-29
+author: dma ([@dma](https://hachyderm.io/@dma))
+description: Results of the first Nivenly member vote
+---
+
+Earlier this month, we sent out the first Nivenly Member vote covering Pachli, 
+Haidra, and general feedback.  As per our bylaws, we need 50% of the membership
+to participate in a vote for quorum, and 50% of total voting members to be in
+consensus for any result to be binding.  As we have 74 members, this means 37
+needed to vote and 18 needed to agree to any individual item.
+
+Unfortunately, only 14 responses were registered in total meaning that the 
+results of the vote are not binding.  However, they do still provide us with a
+direction from the Nivenly members, and that will help inform the decisions the
+Board now makes around these issues.
+
+Thank you very much to everyone that participated in discussions around these 
+projects, and special thanks to the members that voted.
+
+The rest of this post will cover each question in turn with the results, and 
+how we are interpreting the results.
+
+## New project proposals
+
+### Haidra: member project or incubated project?
+
+This was a question we anticipated would be fairly split and indeed the results
+are, with exactly 50% voting "NO" to Haidra being either a regular or incubated
+project, and 50% voting "YES".
+
+7% voted that Haidra should reapply in a year with any concerns mitigated.
```

Comment:
I could see people being confused that it's a 50-50 split, with an additional 7%. I'd clarify something along the lines of "of these votes, 7% had a clarifying note that Haidra could reapply in a year if..."

Reply:
I wanted to avoid the other confusion of "is that 7% of the 50% (ie, 3.5% of the total)?" .. i'll reword